src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+
```

Comment:
Lots of empty lines here - you don't need to put them between every two lines of code :)